spec/requests/dashboard_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe DashboardController, type: :request do
+  describe 'GET /show' do
+    it 'returns http success' do
+      user = { test: 'test' }
+      Utils::Auth.login(user, roles: [])
```

Comment:
❗ After implementing `mock_user`, call it here:
```suggestion
      Utils::Auth.login(mock_user, roles: [])
```